orchagent/routeorch.cpp

```diff
@@ -387,17 +388,17 @@ bool RouteOrch::addNextHopGroup(IpAddresses ipAddresses)
     vector<sai_attribute_t> nhg_attrs;
 
     nhg_attr.id = SAI_NEXT_HOP_GROUP_ATTR_TYPE;
-    nhg_attr.value.s32 = SAI_NEXT_HOP_GROUP_ECMP;
+    nhg_attr.value.s32 = SAI_NEXT_HOP_GROUP_TYPE_ECMP;
     nhg_attrs.push_back(nhg_attr);
 
-    nhg_attr.id = SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_LIST;
+    nhg_attr.id = SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_MEMBER_LIST;
```

Comment:
As Vadym noted, SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_MEMBER_LIST is read only attribute
When creating next hop group, you aren't giving the member list
Instead you create a next hop group
you create next hops
And then to add a next hop to group, you create a next hop group member
So seems the flow here should be 
Create the next hop group without the list
Then, foreach next hop in next_hop_ids, create a next hop group member, given the group id and the next hop id